scripts/visualization_data_converter.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
```

Comment:
nit: can you add these strict settings? https://github.com/awslabs/soci-snapshotter/blob/main/scripts/add-ltag.sh#L17

Here's an explanation of what they do: https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425

Reply:
Added. I'm assuming we need to add these strict settings to the regression check script as well.